src/Microsoft.Performance.SDK.Tests/TestClasses/StubTableProvider.cs

```diff
@@ -22,8 +22,8 @@ public FakeTableProvider()
 
         public IEnumerable<TableDescriptor> Discover(ITableConfigurationsSerializer tableConfigSerializer)
         {
-            this.DiscoverCalls.Add(tableConfigSerializer);
-            return this.DiscoverReturnValue;
+            DiscoverCalls.Add(tableConfigSerializer);
```

Comment:
Sorry, was trying to also refer to these two lines as well in my previous comment :)